code/modules/antagonists/villain/werewolf/werewolf_transformation.dm

```diff
@@ -15,105 +16,92 @@
 					H.flash_fullscreen("redflash3")
 					transforming = world.time // timer
 
-	// Begin transformation
-	else if(transforming)
-		if (world.time >= transforming + 35 SECONDS) // Stage 3
-			H.werewolf_transform()
-			transforming = FALSE
-			transformed = TRUE // Mark as transformed
-
-		else if (world.time >= transforming + 25 SECONDS) // Stage 2
-			H.flash_fullscreen("redflash3")
-			H.emote("agony", forced = TRUE)
-			to_chat(H, span_userdanger("UNIMAGINABLE PAIN!"))
-			H.Stun(30)
-			H.Knockdown(30)
-
-		else if (world.time >= transforming + 10 SECONDS) // Stage 1
-			H.emote("")
-			to_chat(H, span_warning("I can feel my muscles aching, it feels HORRIBLE..."))
-
-
-	// Werewolf reverts to human form during the day
-	else if(transformed)
-		H.real_name = wolfname
-		H.name = wolfname
-
+	if(!transformed)
+		// Begin transformation
+		if(transforming)
+			if (world.time >= transforming + 35 SECONDS) // Stage 3
+				for(var/obj/item/item as anything in H.get_equipped_items(FALSE))
+					if(istype(item, /obj/item/clothing))
+						item.take_damage(damage_amount = item.max_integrity * 0.4, sound_effect = FALSE)
+					else if(istype(item, /obj/item/weapon) || istype(item, /obj/item/storage))
+						H.dropItemToGround(item, silent = TRUE)
+
+				var/mob/living/carbon/human/species/werewolf/new_werewolf = generate_werewolf(H)
+				new_werewolf.apply_status_effect(/datum/status_effect/shapechange_mob/die_with_form, H, FALSE)
+				new_werewolf.adjust_skillrank(/datum/skill/combat/wrestling, 4, TRUE)
```

Comment:
this should probably be kept at 5 or port the "hard to grab" trait from azure to prevent RK/captain/monk/orcs from grab-trolling the giant werewolf